source/model/Database.py

```diff
@@ -0,0 +1,24 @@
+class Database():
```

Comment:
I suppose this is a singleton, but please also describe this in the documentation above the class. Otherwise you would have the confusion as to why you are creating a database multiple times in the code.